roles/bastion/defaults/main.yml

```diff
@@ -12,11 +15,8 @@ bastion_users:
   - rmoe
   - bradonk
   - doc
-  - cideploy
   - jkeating
   - jesusaur
```

Comment:
You're down here in remove_users. Are we doing a follow-up commit to add back those users who need it to bastion_users?